lib/yform/value/upload.php

```diff
@@ -52,7 +52,7 @@ public function enterObject()
         $errors = [];
 
         // Hat Formular einen Uniquekey für dieses Formular und dieses Feld übertragen ?
-        $unique = $this->params["this"]->getFieldValue($this->getId(), 'unique');
+        $unique = $this->params["this"]->getFieldValue($this->getId(), 'unique', 'importfile');
```

Comment:
Ich habe von der Upload-Klasse wenig Ahnung, aber mir kommt es komisch vor, dass in der allgemeinen Upload-Value-Klasse nun etwas von "importfile" steht. Der Import ist doch nur ein Fall, wo die Upload-Klasse verwendet wird?